slack_sdk/web/async_client.py

```diff
@@ -21,6 +22,10 @@
     _update_call_participants,
     _warn_if_text_or_attachment_fallback_is_missing,
     _remove_none_values,
+    _to_v2_upload_file_item,
```

Comment:
@seratch - very minor but this could then be named to _to_v2_file_upload_item for consistency

Reply:
@srajiang Thanks for catching this, renamed ✅